src/CallbackFilterHandler.php

```diff
@@ -114,7 +150,7 @@ public function handle(array $record)
         if (!$this->handler instanceof HandlerInterface) {
             $this->handler = call_user_func($this->handler, $record, $this);
```

Comment:
Is there any objective reason why you don't like the `call_user_func` syntax ?

Reply:
it's a tiny bit faster (https://wildwolf.name/php-call_user_func_array-vs-dynamic-call/) and it may look simpler when `$this->handler` is extracted to a variable

Reply:
> Is there any objective reason why you don't like the `call_user_func` syntax ?

Not really. I only think, its easier to read.

Reply:
@weirdan really tiny then ;-)
BTW thanks for the link

Reply:
@mimmi20 👍 to be easier to read. Will apply change tomorrow

Reply:
commit 8d4f99ca62ae3df6a7b609fdf68c05af6bf62dda resolved it